package-lock.json

```diff
@@ -1,7858 +0,0 @@
-{
-  "name": "testcafe-browser-provider-lambdatest",
```

Comment:
This should also be included in the repository.

Reply:
Included package-lock.json, please check now